### ISSUE TITLE
Use trip advisor as a start

### ISSUE THREAD

**Issue by saxenanurag** [CLEpy/CLEpy-Guide-to-Cleveland#1]
This can be a good guide: https://www.tripadvisor.com/Attractions-g50207-Activities-Cleveland_Ohio.html

**Comment by tylerdave** [comment]
Yes. I think there are a ton of good resources out there! I'd really like to create our own content so we can point to what we think matters and then link to resources like this for further detail.

I've added some basic guidelines on the wiki so you can see what I mean: https://github.com/CLEpy/CLEpy-Guide-to-Cleveland/wiki

**Comment by saxenanurag** [comment]
Is this acceptable: https://github.com/CLEpy/CLEpy-Guide-to-Cleveland/wiki/Food-and-Drink
?
